.github/workflows/tag_latest_image.yaml

```diff
@@ -1,10 +1,11 @@
 ---
-name: Tag Prod Latest
-'on':
+name: Tag Latest Image
+on:
   pull_request:
     branches:
-    - master
     - main
+    - master
+    - develop
```

Comment:
add in the `develop` branch as the GHA for that branch does exactly the same thing